src/SprykerSdk/Spryk/SprykFacadeInterface.php

```diff
@@ -15,10 +15,18 @@ interface SprykFacadeInterface
      * @param string $sprykName
      * @param string[] $includeOptionalSubSpryks
      * @param \SprykerSdk\Spryk\Style\SprykStyleInterface $style
+     * @param string|null $targetModuleName
+     * @param string|null $dependentModuleName
      *
      * @return void
      */
-    public function executeSpryk(string $sprykName, array $includeOptionalSubSpryks, SprykStyleInterface $style): void;
+    public function executeSpryk(
+        string $sprykName,
+        array $includeOptionalSubSpryks,
+        SprykStyleInterface $style,
+        ?string $targetModuleName = '',
```

Comment:
Shouldnt this default to null then? Or why empty string vs null as two different empty values?